viewer/src/index.ts

```diff
@@ -4,12 +4,17 @@
 
 // Everything that is exposed to the user should be defined here
 
+export { RevealManager } from './public/RevealManager';
+// TODO 2020-05-15 larsmoa: CadNode is _internal_ and should not be exported
+export { CadNode } from './dataModels/cad/internal/CadNode';
+export { ModelNodeAppearance } from './dataModels/cad/internal/ModelNodeAppearance';
+
 // CAD
 export { CadModel, loadCadModelFromCdf, loadCadModelByUrl } from './dataModels/cad/internal';
 export { SectorMetadata, SectorModelTransformation } from './dataModels/cad/internal/sector/types';
 export { CadRenderHints } from './dataModels/cad/public/CadRenderHints';
 export { CadLoadingHints } from './dataModels/cad/public/CadLoadingHints';
-
+export { intersectCadNodes } from './dataModels/cad/internal/picking';
```

Comment:
Is there any reason why this one is not using alias?

Reply:
Not really.